master/networking/service-advertisement.md

```diff
@@ -58,6 +59,9 @@ In order to implement this behavior, Calico does the following.
    cluster IP for that service is advertised as a /32 route from the
    nodes that have endpoints for that service.
 
+>**Note**: in order to set `externalTrafficPolic: Local` on a service, the service be of type `LoadBalancer` or `NodePort`.
```

Comment:
**Note**: To set `externalTrafficPolic: Local` on a service, the service must be type `LoadBalancer` or `NodePort`.